styles/main.scss

```diff
@@ -161,6 +161,26 @@ button {
     margin-top: 6px;
   }
 }
+
+.yc-btn-secondary--large {
+  border: 1px solid var(--yc-primary-color);
+  color: var(--yc-primary-color);
+  padding: 14px 140px;
```

Comment:
why 140px for padding 
<img width="392" alt="Screen Shot 2023-02-27 at 12 30 47" src="https://user-images.githubusercontent.com/93322743/221552915-9891a5e8-59a4-4062-8e7a-6850ab5a40da.png">
 :/ ?